frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -17,70 +17,203 @@ describe("scenarios > question > filter", () => {
+        cy.log("**-- 2. Create nested card based on the first one --**");
+
+        cy.request("POST", "/api/card", {
+          name: "Q2",
+          dataset_query: {
+            database: 1,
+            query: { "source-table": `card__${Q1_ID}` },
+            type: "query",
+          },
+          display: "table",
+          visualization_settings: {},
+        }).then(({ body: { id: Q2_ID } }) => {
+          cy.log("**-- 3. Create a dashboard --**");
+
+          cy.request("POST", "/api/dashboard", {
+            name: "12985D",
+          }).then(({ body: { id: DASHBOARD_ID } }) => {
+            cy.log("**-- 4. Add 2 filters to the dashboard --**");
+
+            cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}`, {
+              parameters: [
+                {
+                  name: "Date Filter",
+                  slug: "date_filter",
+                  id: "78d4ba0b",
+                  type: "date/all-options",
+                },
+                {
+                  name: "Category",
+                  slug: "category",
+                  id: "20976cce",
+                  type: "category",
+                },
+              ],
+            });
 
-    // Add Q2 to a dashboard
-    cy.findByText("Yes please!").click();
-    cy.findByText("Orders in a dashboard").click();
+            cy.log("**-- 5. Add nested card to the dashboard --**");
 
-    // Add two dashboard filters
-    cy.get(".Icon-filter").click();
-    cy.findByText("Time").click();
-    cy.findByText("All Options").click();
-    cy.findAllByText("Select…")
-      .last()
-      .click();
-    cy.findByText("Created At").click();
+            cy.request("POST", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+              cardId: Q2_ID,
+            }).then(({ body: { id: DASH_CARD_ID } }) => {
+              cy.log(
+                "**-- 6. Connect dashboard filters to the nested card --**",
+              );
 
-    cy.get(".Icon-filter").click();
-    cy.findByText("Other Categories").click();
-    cy.findAllByText("Select…")
-      .last()
-      .click();
-    popover().within(() => {
-      cy.findByText("Category").click();
+              cy.request("PUT", `/api/dashboard/${DASHBOARD_ID}/cards`, {
+                cards: [
+                  {
+                    id: DASH_CARD_ID,
+                    card_id: Q2_ID,
+                    row: 0,
+                    col: 0,
+                    sizeX: 10,
+                    sizeY: 8,
+                    series: [],
+                    visualization_settings: {},
+                    // Connect both filters and to the card
+                    parameter_mappings: [
+                      {
+                        parameter_id: "78d4ba0b",
+                        card_id: Q2_ID,
+                        target: [
+                          "dimension",
+                          ["field-id", PRODUCTS.CREATED_AT],
+                        ],
+                      },
```

Comment:
Remove?

Reply:
I don't get it.
This is the original repro v1, just re-written to use API. Why would you remove it and change repro steps now?

Reply:
Then leave it